markdown/dev/reference/api/config/options/en.md

```diff
@@ -7,30 +7,30 @@ file give designers flexility to make one pattern with different variations.
 
 ## The use case for (design) options
 
-One of the things that sets FreeSewing apart is that sewing patterns are not 
-static. Each pattern is generated on the spot to accommodate the input 
+One of the things that sets FreeSewing apart is that sewing patterns are not
+static. Each pattern is generated on the spot to accommodate the input
 provided by the user. Input that typically includes their measurments.
 
-This *made-to-measure* approach is sort of *our thing* at FreeSewing, 
+This *made-to-measure* approach is sort of *our thing* at FreeSewing,
 but why stop there?
-There's a lot of things that can be left up to the user and taken into 
+There's a lot of things that can be left up to the user and taken into
 consideration when drafting the pattern. Things like how many buttons to use,
 whether or not to include pockets, shape of the collar, and so on. The only
 limit really is the creativity of the designer.
 
-The `options` section in a pattern's configuration file is what makes this 
+The `options` section in a pattern's configuration file is what makes this
 possible.
 
 ## The five option types you should know
 
 There are the five option types that an aspiring pattern designer should be
 familiar with:
 
-1. [**boolean** options][bool] are for yes/no choices
-1. [**counter** options][count] are for integer values
-1. [**degree** options][deg] are for degrees
-1. [**list** options][list] are for a list of possible choices
-1. [**percentage** options][pct] are for percentages
+1.  [**boolean** options][bool] are for yes/no choices
```

Comment:
Linter auto updated the numbers, this again may be worth checking if it renders correctly in the site

Reply:
Renders correctly.